<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>storedViewDialogForm</class>
 <widget class="QWidget" name="storedViewDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>359</height>
   </rect>
  </property>
  <property name="font">
   <font/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBar" name="titleBar">
     <property name="title">
      <string>Stored views</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="viewContent">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QListView" name="listView"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonLoadView">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Load view</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonDeleteView">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Delete view</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonAddView">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Add current view</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="useDateCheckBox">
             <property name="text">
              <string>Load/store with date</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="0">
            <widget class="QLineEdit" name="lineEditTitle">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Title:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Description:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="CustomTextEdit" name="textEditDescription"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QFrame</extends>
   <header>Dialog.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomTextEdit</class>
   <extends>QTextEdit</extends>
   <header>StoredViewDialog_p.hpp</header>
  </customwidget>
  <customwidget>
   <class>StelCloseButton</class>
   <extends>QPushButton</extends>
   <header>gui/StelCloseButton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
